portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -31,4 +40,10 @@ definition {
 			value1 = "Displaying Form: ${formName} (Modified)");
 	}
 
+	macro viewTranslationLanguage {
+		AssertElementPresent(
+			changeLanguageTo = "${language}",
```

Comment:
@wesleyroberts, please rename:

- `changeLanguageTo` to `key_language`

Reply:
changes made